src/main/resources/db/changelog/changes/vine-changelog-2023.02.28.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+        http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+    <changeSet id="1" author="volodymyr">
+        <createTable tableName="vine">
+            <column name="id" type="varchar(36)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="manufacturer" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="grape_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="color" type="varchar(36)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="is_sparkling" type="tinyint(4)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="sugar" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="country" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="region" type="varchar(255)"/>
+            <column name="year" type="int(4)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="price" type="float(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="amount" type="int(5)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="abv" type="float(5)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="post_office">
+            <column name="id" type="varchar(36)">
```

Comment:
Do u want to use UUID here?

Reply:
yes

Reply:
in that way it is better to use BINARY type instead of VARCHAR. You can read a little bit more about storing uuid values

Reply:
will change in this place and other (where store id) to binary(16)